src/backend/mysql/table.rs

```diff
@@ -78,7 +78,19 @@ impl TableBuilder for MysqlQueryBuilder {
                 ColumnType::Date => "date".into(),
                 ColumnType::Interval(_, _) => "unsupported".into(),
                 ColumnType::Binary(length) => match length {
-                    Some(length) => format!("binary({})", length),
+                    Some(length) => {
+                        if *length <= 255 {
+                            format!("binary({})", length)
+                        } else if *length <= 65535 {
+                            //64KiB
+                            "blob".into()
+                        } else if *length <= 16777215 {
+                            //16MiB
+                            "mediumblob".into()
+                        } else {
+                            "longblob".into() //longblob max to 4294967295 4GiB but this limit can be config
+                        }
+                    }
```

Comment:
Hey @hunjixin, thank you! Just a nitpick, wouldn't it be more readable to express the number as power of 2?

```suggestion
                    Some(length) => {
                        if *length < (2 << 8) {
                            format!("binary({})", length)
                        } else if *length < (2 << 16) {
                            //64KiB
                            "blob".into()
                        } else if *length < (2 << 24) {
                            //16MiB
                            "mediumblob".into()
                        } else {
                            //longblob max to 4294967295 4GiB but this limit can be config
                            "longblob".into()
                        }
                    }
```

References:
- https://dev.mysql.com/doc/refman/8.0/en/storage-requirements.html#data-types-storage-reqs-strings
- https://mariadb.com/kb/en/blob/

Reply:
how does that suggest change actually works?

Reply:
Select the lines you want to comment and suggest changes. Then, type the suggested changes in code block of `suggestion` language. See [GitHub Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request)